matlab-src/mex-wholebodymodel_balancing/forwardDynamics.m

```diff
@@ -43,41 +40,32 @@
 %% fixing the world reference frame
 qT         = [x_b; qt_b];
 [~,R_b]    = frame2posrot(qT);
- 
-wbm_setWorldFrame(R_b,x_b,[0 0 -9.81]');
 
-wbm_updateState(qj,dqj,[dx_b;omega_W]);
-
-%% MexWholeBodyModel calls
-% warning: there's an issue with the optimized mode: for now, it won't be
+%% mexWholeBodyModel calls
+% warning: there's an issue with the wbm optimized mode: for now, it won't be
 % used
-% M = wbm_massMatrix(); 
-% h = wbm_generalisedBiasForces();
-% H = wbm_centroidalMomentum();
-
-R_binv = eye(3)/R_b;
+R_b_tr = R_b';
 
-M      = wbm_massMatrix(R_binv,x_b,qj); 
-h      = wbm_generalisedBiasForces(R_binv,x_b,qj,dqj,[dx_b;omega_W]);
-H      = wbm_centroidalMomentum(R_binv,x_b,qj,dqj,[dx_b;omega_W]);
+M      = wbm_massMatrix(R_b_tr,x_b,qj); 
```

Comment:
I noticed this bug some time ago, but I tought it was due to WBM_functions structure and I didn't open an issue before.